### PR TITLE
Added ddg syntax cheatsheet (Fixes #756)

### LINKED ISSUE #756: DuckDuckGo Cheatsheet
As mentioned by a community member, it might be useful to copy some of the great info we have available [here](https://duck.co/help/results/syntax) and elsewhere in the help pages, into a cheat sheet.

### PR DESCRIPTION
**Pull request by gautamkrishnar**
Fixes #756 

**What does your Instant Answer do?**
Display DDG Syntax

**What is the data source for your Instant Answer? (Provide a link if possible)**
https://duck.co/help/results/syntax

**Why did you choose this data source?**
mentioned in the issue

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
All DDG lovers

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
ddg2

---

IA Page: https://duck.co/ia/view/duckduckgo_syntax_cheat_sheet

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
@mintsoft @moollaza

**Comment by jagtalon** [comment]
@gautamkrishnar thank you! Checking right now.

File: share/goodie/cheat_sheets/json/duckduckgo-syntax.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  ],
+  "sections": {
+    "Triggers": [
+      {
+        "key": "LeBron James news",
+        "val": "Add, \"news\" to your searches to generate instant-answer news results"
+      },
+      {
+        "key": "Philadelphia map",
+        "val": "Add, \"map\" to your search to generate instant-answer map results"
```
Comment (jagtalon) [inline share/goodie/cheat_sheets/json/duckduckgo-syntax.json]: Remove the "," so that it's just "Add "map" to your search ..."

Comment (jagtalon) [inline share/goodie/cheat_sheets/json/duckduckgo-syntax.json]: "instant-answer" should be "Instant Answer"

Comment (jagtalon) [inline share/goodie/cheat_sheets/json/duckduckgo-syntax.json]: I just realized that this was just copied from the duck.co page. My bad this is good.

**Comment by gautamkrishnar** [comment]
@jagtalon :+1:

**Comment by jagtalon** [comment]
:+1:

**Comment by gautamkrishnar** [comment]
Thanks for editing It. :smile: :wink: @jagtalon

File: share/goodie/cheat_sheets/json/duckduckgo-syntax.json [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      {
+        "key": "Apple AND Mango",
+        "val": "Every search term should be used by default. That is, we try hard not to autocorrect your query. In other words, we treat your terms as if you typed AND in between them"
+      },
+      {
+        "key": "Apple OR Mango",
+        "val": "If you want to include one term or another, use the uppercase keyword OR in between terms. OR will only operate on adjacent words. Foo bar OR baz is equivalent to Foo ((bar)OR(baz))."
+      },
+      {
+        "key": "\"Eat food\" OR \"Drink water",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/duckduckgo-syntax.json]: Missing a closeing `\"` here

File: share/goodie/cheat_sheets/json/duckduckgo-syntax.json [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    ],
+    "Safe search": [
+      {
+        "key": "",
+        "val": "Add !safeoff to the end of your search to turn off safe search for that search"
+      }
+    ],
+    "Regional search": [
+      {
+        "key": "",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/duckduckgo-syntax.json]: I don't like the idea of empty `key`'s. Let's be consistent: We should have examples in all the key's or at least a "title" that's indicates what the `val` is about

**Comment by gautamkrishnar** [comment]
Created new PR to fix the issues

**Comment by moollaza** [comment]
@gautamkrishnar congrats! It's live: https://duckduckgo.com/?q=duckduckgo+syntax+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by gautamkrishnar** [comment]
:+1:

**Pull request merged**